docs: update AI Studio core concepts (#1729)
TykTechnologies/tyk-docs: ai-management/ai-studio/core-concepts.mdx

### User
Represents an individual interacting with Tyk AI Studio, managed within the [User Management system](/ai-management/ai-studio/user-management#core-concepts).

### Team

> Replaces or updates the outdated "Group" concept. Teams are the central access control mechanism to organize users and manage their access to LLMs, data sources, and tools via Catalogs. See also [Teams](/ai-management/ai-studio/teams).

Reviewer: ```suggestion
Teams are the central access control mechanism for organizing users and managing their access to LLMs, data sources, and tools via Catalogs. See also [Teams](/ai-management/ai-studio/teams).
```

### Chat / Chat Room
Customized interfaces configured by admins that combine specific LLMs, tools, data sources, and system prompts for user interactions. See also [Chats](/ai-management/ai-studio/chats).

### LLM Call Settings
Default runtime parameters (e.g., temperature, max tokens, top P) for LLMs used in chat interactions and middleware function calls. See also [LLM Call Settings](/ai-management/ai-studio/call-settings).

### Model Price
Defines the cost per million tokens (input, output, cache) for using different language models, essential for analytics and budget enforcement. See also [Model Prices](/ai-management/ai-studio/model-prices).

### Plugin
The extensibility system using a Unified Plugin SDK to add custom capabilities like Edge Gateway middleware, UI extensions, Agent plugins, and Object Hooks. See also [Plugins](/ai-management/ai-studio/plugins).

### Budget
Defines hard limits on costs (amount, currency, period) applied at the organization or LLM configuration level. See also [Budget Control](/ai-management/ai-studio/budgeting).

### Telemetry

> System that collects anonymized usage statistics to help improve the product. See also [Telemetry](/ai-management/ai-studio/telemetry).

Reviewer: ```suggestion
```
